redbox-core/redbox/loader/loaders.py

```diff
@@ -135,77 +62,30 @@ def _chunking(self) -> Any:
         if response.status_code != 200:
             raise ValueError(response.text)
 
-        return response.json()
+        return response.json() or []
 
-    def extract_metadata(self):
+    def extract_metadata(self) -> dict:
         """
         Extract metadata from first 1_000 chunks
         """
 
-        elements = self._chunking()
-        if not elements:
-            self.metadata = self.default_metadata
-
-        else:
-            # Get first 1k tokens of processed document
-            first_n = self.get_first_n_tokens(elements, 1_000)
+        chunks = self._chunking()
+        first_thousand_words = "".join(chunk["text"] for chunk in chunks)[:1_000]
+        metadata = self.create_file_metadata(first_thousand_words)
+        return GeneratedMetadata.model_validate(metadata).model_dump()
 
-            # Get whatever metadata we can from processed document
-            doc_metadata = self.get_doc_metadata(chunks=elements, n=3, ignore=None, max_size=524288 - len(first_n))
```

Comment:
we've dropped getting the metadata from unstructured, was that intentional? This metadata fed into the prompt before too which is how we could get document name for example

Reply:
> was that intentional

yep, because in `lazy_load` the unstructured-metadata is regenerated anyway, and the llm-generated-metadata is explicitly handled (i.e. the merging of the llm and unstructured metadata never did anything in the first place)

Reply:
but wasn't the combined metadata bringing the filename in from unstructured metadata? The LLM generated stuff can't find that now?

Reply:
yes, but then it was ignored
it also always gets regenerated every time we use unstructured

Reply:
i see your point, how about [6bcf9be](https://github.com/i-dot-ai/redbox/pull/1147/commits/6bcf9be62b857735702f687005ed9c0834f664d2) ?